### PR TITLE
Fix/ik/image retrieval demo

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
@IRDonch

**Comment by IRDonch** [comment]
Jenkins: test this please

File: demos/python_demos/image_retrieval_demo/image_retrieval_demo/image_retrieval.py [hunk 1]
```diff
@@ -35,16 +35,15 @@ def __init__(self, model_path, device, cpu_extension):
...
         self.exec_net = ie.load_network(network=self.net, device_name=device)
 
     def predict(self, image):
         ''' Takes input image and returns L2-normalized embedding vector. '''
 
         assert len(image.shape) == 4
         image = np.transpose(image, (0, 3, 1, 2))
-        out = self.exec_net.infer(inputs={'Placeholder': image})[
-            'model/tf_op_layer_mul/mul/Normalize']
-        out = out / np.linalg.norm(out, axis=-1)
```
Comment (IRDonch) [inline demos/python_demos/image_retrieval_demo/image_retrieval_demo/image_retrieval.py]: Was this normalization always unnecessary? Or did MO convert the network incorrectly in previous releases?

Comment (Ilya-Krylov) [reply demos/python_demos/image_retrieval_demo/image_retrieval_demo/image_retrieval.py]: Just checked old model and R3. Looks like it always was unnecessary

**Pull request merged**